src/components/card/card-dialog/CardDialog.js

```diff
@@ -110,7 +110,7 @@ CardDialog.propTypes = {
   addCard: PropTypes.func,
   updCard: PropTypes.func,
   card: PropTypes.shape({
-    id: PropTypes.string,
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Does not work, still wants id as a string (number comes from api)

Reply:
You forgot to add those types to CardItem component

Reply:
Exactly!